emukit/examples/profet/train_meta_model.py

```diff
@@ -0,0 +1,193 @@
+import argparse
+import json
+import os
+import pickle
+from copy import deepcopy
+import tarfile
+from urllib.request import urlretrieve
+import GPy
+import numpy as np
+import torch
+from GPy.models import BayesianGPLVM
+from pybnn.bohamiann import Bohamiann
+
+from emukit.examples.profet.meta_benchmarks.architecture import get_default_architecture_classification, get_default_architecture_regression, get_default_architecture_cost
+from emukit.examples.profet.meta_benchmarks.meta_forrester import get_architecture_forrester
+
+
+def download_data(path, source='http://www.ml4aad.org/wp-content/uploads/2019/05/profet_data.tar.gz'):
+
+    l = urlretrieve(source)[0]
+
+    tar = tarfile.open(l)
+    tar.extractall(path=path)
+    tar.close()
+
+
+def load_data(path, filename):
+    res = json.load(open(os.path.join(path, filename), "r"))
+    return np.array(res["X"]), np.array(res["Y"]), np.array(res["C"])
+
+
+def normalize_Y(Y, indexD):
+    max_idx = np.max(indexD)
+    Y_mean = np.zeros(max_idx + 1)
+    Y_std = np.zeros(max_idx + 1)
+    for i in range(max_idx + 1):
+        Y_mean[i] = Y[indexD == i].mean()
+        Y_std[i] = Y[indexD == i].std() + 1e-8
+        Y[indexD == i] = (Y[indexD == i] - Y_mean[i]) / Y_std[i]
+    return Y, Y_mean[:, None], Y_std[:, None]
+
+
+if __name__ == "__main__":
```

Comment:
If you could wrap code inside this `if` into a high level API, it would be easy to use Profet both as a lib and from command line. It is a huge chunk of code any way